functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -39,9 +46,10 @@
     private File workingDirectory;
     private String mavenOpts = "-Xms512m -Xmx1024m";
 
-    BuildConfigurator(ProjectBuilder projectBuilder) {
+    BuildConfigurator(ProjectBuilder projectBuilder, Path root) {
```

Comment:
the `ProjectBuilder` contains the information about root directory, so it should be possible to get it using getter method.